smoke-test.sh

```diff
@@ -0,0 +1,34 @@
+# grab go-fdo-client at revision (TODO(runcom): stick to a revision)
+git clone https://github.com/fido-device-onboard/go-fdo-client
+cd go-fdo-client
+go build -o fdo_client ./cmd/fdo_client
+cd ../
+rm -rf mfg.db rv.db own.db
+# start manufacturing server
+go-fdo-server serve 127.0.0.1:8038 --db ./mfg.db --db-pass 1234Cia0@! --debug &
```

Comment:
I think it would be appropriate for this PR to add flags to serve to enable/disable protocols so that each server in the smoke test can be configured to only do DI, TO0/TO1, and TO2.